libs/popup_manager.py

```diff
@@ -28,6 +28,9 @@ def __init__(self, proxy):
         self.signature_index = 0
         self.current_parameter = 0
 
+        # Used to avoid looping
+        self.suppress_onhidden = 0
```

Comment:
Do we use the ``suppress_onhidden`` for the condition of anything? Or is it just for book keeping?